ocean_provider/routes.py

```diff
@@ -343,7 +343,7 @@ def initialize():
 
 
 @services.route('/download', methods=['GET'])
-def download():
+def access():
```

Comment:
might have missed some discussion for this change but download !== access. Giving access or what we usually call `initialize` happens for both cases, download & compute. So it seems more logical to keep this name as `download`

Reply:
I asked @alexcos20 about the task and he told me that the access service is related to download route so... That's why I modified and @ssallam told me to change the functions' names in some previous commets here. Let me know what should I modify.

Reply:
I would keep the `download` name as it directly describes what it does.

Reply:
So then should I keep 'access' or download is enough?

Reply:
well, we found a conceptual flaw because of legacy. In this context it really makes sense to keep the name as `download` but since the beginning the service type in the DDO is called access service, hence the confusion. It should be named `download` too but I don't think we'll do that change

Reply:
Agreed, keep it as download

Reply:
Ok, thank you for the information. I will keep 'download'